app/dao/notification_history_dao.py

```diff
@@ -33,3 +35,18 @@ def delete_notification_history_older_than_datetime(older_than_datetime, query_l
     current_app.logger.info(
         "Deleted %s rows from notification_history older than %s", total_deleted, older_than_datetime
     )
+
+
+def delete_notification_history_between_two_datetimes(start: datetime, end: datetime):
```

Comment:
💀 just tested this locally and it does end up getting called with strings. when we call `.apply_async` with the datetimes as args, celery calls `isoformat` (or something similar) on them automatically. But it doesn't reverse that on the way out. So probably worth updating this to `str` actually 🥲

Reply:
Thanks Sam. I've removed the type hints for now. What I'm going to do is merge this and get it tested in prod and then going to add back in the type hints this afternoon (and adapt my tests to also being receiving strings) when I have no meetings!